contributors/devel/sig-architecture/validation-suites.md

```diff
@@ -0,0 +1,82 @@
+# Validation Suite Testing in Kubernetes
+
+A Kubernetes Validation suite is a subset of e2e tests that are used to
+validate correct functionality of an optional capability of Kubernetes. For example,
+Validation suites may be used to verify that optional storage or networking 
```

Comment:
So this language overlaps with "Profiles".  We should probably refine the taxonomy.

Reply:
@timothysc Good point. I will revise and also add  some clarifying sentences that explain how validation suites are different from profiles and the flexibility they provide us to create suites of bucketized tests to provide validation independent of the process of defining/deciding that the validation suite should eventually become merged into the base profile, become their own profle etc.

Reply:
+1